modules/nf-core/cellranger/mkref/tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+process {
+    withName: CELLRANGER_MKREF {
+        publishDir = [ path: { "output" } ]
```

Comment:
do we need to publish?

Reply:
Probably not 😉

Reply:
then let's delete the file

Reply:
actually we're not publishing in tests, so yeah, let's delete